plugins/channelrx/freqscanner/freqscannersink.cpp

```diff
@@ -27,7 +27,6 @@
 #include "freqscannersink.h"
 
 FreqScannerSink::FreqScannerSink(FreqScanner *ilsDemod) :
-        m_freqScanner(ilsDemod),
```

Comment:
Remove from constructor parameters if not needed

Reply:
ok